src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<p>For all plots: there should be axis labels (especially for the last one).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p><strong>Important</strong>: matplotlib is currently declared as dev dependency, since we use plotly for our plots. We could move it to the user dependencies, but I'd rather use plotly here to keep consistency and to avoid requiring an additional dependency to users.</p>

Reply:
a492fccf